docs/guides/fine_tuning.md

```diff
@@ -1,3 +1,20 @@
-# Fine-Tuning
+# Fine Tuning API
+The Fine Tuning API allows you to fine tune various open-source LLMs on your own data, then make inference calls to the resulting LLM.
 
-Coming soon!
+## Preparing Data
+Your data must be formatted as a CSV file that includes two columns: `prompt` and `response`. The data needs to be uploaded to somewhere publicly accessible, so that we can read the data to fine tune on it. For example, you can upload your data to a public s3 bucket.
+
+## Launching the Fine Tune
+Once you have uploaded your data, you can use our API to launch a Fine Tune. You will need to provide the base model to train off of, the locations of the training and validation files, an optional set of hyperparameters to override, and an optional suffix to append to the name of the fine tune. 
+
+Currently, we support the following base models:
+
+- `mpt-7b-instruct`
```

Comment:
Might be worth linking to the Model Zoo page (you can do something like `[Model Zoo](/model_zoo)` to do a markdown link to it). That way we don't have to worry about things getting out of sync.